CMakeLists.txt

```diff
@@ -217,6 +217,9 @@ endif()
 
 if(ENABLE_PIE)
 	add_definitions(-fPIE)
+	if(CMAKE_SYSTEM_NAME MATCHES FreeBSD)
+		add_definitions(-fPIC)
```

Comment:
I added this because it was giving errors about relocations in libcore.a not being compatible with shared objects. It's presumably a bug in FreeBSD's gcc, but it's a bug that needs to be dealt with.